### PR TITLE
update to MonoAndroid8.1 and Xamarin.Forms 27.0.2 

### PR DESCRIPTION
**Pull request by jennyf19**
update adal nuget package to include core dll's
add missing XML comment in PlatformParameters for embeddedWebview boolean

- addressing [issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/553)

### CODE REVIEW

**Comment by jennyf19** [comment]
The VSTS build will fail because the file paths are set for MonoAndroid7

File: adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/PlatformParameters.cs [hunk 1]
```diff
@@ -98,6 +98,10 @@ public PlatformParameters(Activity callerActivity, bool useBroker, PromptBehavio
         /// </summary>
         public PromptBehavior PromptBehavior { get; set; }
 
+        /// <summary>
+        /// Used to set embedded 
+        /// ADAL only supports embedded webview.
```
Comment (henrik-me) [inline adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/PlatformParameters.cs]: >ADAL only supports embedded webview. [](start = 12, length = 36)

Should perhaps be in the <remarks>section of the comments instead?

File: adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/PlatformParameters.cs [hunk 2]
```diff
@@ -98,6 +98,10 @@ public PlatformParameters(Activity callerActivity, bool useBroker, PromptBehavio
         /// </summary>
         public PromptBehavior PromptBehavior { get; set; }
 
+        /// <summary>
+        /// Used to set embedded
```
Comment (henrik-me) [inline adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/PlatformParameters.cs]: >Used to set embedded  [](start = 12, length = 21)

half sentence?

**Comment by henrik-me** [comment]
>     <Configurations>Debug;Release;Appveyor</Configurations>

remove?

---
Refers to: adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory.Platform/Microsoft.IdentityModel.Clients.ActiveDirectory.Platform.csproj:8 in e77d554. [](commit_id = e77d554b01ad5f95f9859d3765170dc07cb74b95, deletion_comment = False)

File: msal/src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj [hunk 3]
```diff
@@ -154,12 +154,12 @@
     <None Update="Platforms\net45\WindowsFormsWebAuthenticationDialogBase.cs" SubType="Form" />
     <None Update="Platforms\net45\SilentWindowsFormsAuthenticationDialog.cs" SubType="Form" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == 'MonoAndroid7'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'MonoAndroid8.1'">
```
Comment (henrik-me) [inline msal/src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj]: ><ItemGroup [](start = 2, length = 10)

minor: content of this item group could be moved up to the android item group above

File: adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/PlatformParameters.cs (hunk 2 above)
Comment (jennyf19) [reply adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/PlatformParameters.cs]: yes...that's what happens when trying to finish before the forced update :)

---
In reply to: [190771933](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1069#discussion_r190771933) [](ancestors = 190771933)

File: adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/PlatformParameters.cs (hunk 1 above)
Comment (jennyf19) [reply adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/PlatformParameters.cs]: done.

---
In reply to: [190771904](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1069#discussion_r190771904) [](ancestors = 190771904)

**Comment by jennyf19** [comment]
>     <Configurations>Debug;Release;Appveyor</Configurations>

done.

---
In reply to: [391914636](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1069#issuecomment-391914636) [](ancestors = 391914636)

---
Refers to: adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory.Platform/Microsoft.IdentityModel.Clients.ActiveDirectory.Platform.csproj:8 in e77d554. [](commit_id = e77d554b01ad5f95f9859d3765170dc07cb74b95, deletion_comment = False)

**Comment by jennyf19** [comment]
Sounds good. I agree w/updating Xamarin.Forms as well. Will make a task for this.

**Pull request merged**